src/components/map-marker/readme.md

```diff
@@ -0,0 +1,25 @@
+# gx-map-marker
+
+This element represents a marker inside a `<gx-map>` element.
+`coords` attribute must always have a value. If there is not any value set, the marker will be initialize with default values and it will be drawn at `0, 0` coordinates.
```

Comment:
I do not understand the suggestion, the comment should be like:

`coords` attribute must always have a value. If value is **not** specified, the marker will be initialized with default values and it will be drawn at `0, 0` coordinates.

Reply:
Ready then!